static/progressive-web-app.js

```diff
@@ -14,75 +14,87 @@ const urlsToCache = [
 // Install event (caches all necessary files)
 self.addEventListener("install", (event) => {
```

Comment:
Async this instead

Reply:
like this? `self.addEventListener("install", async (event) => {` not sure why though